internal/checker/cloudformation/rules/logging/verify_s3_bucket_sse_enabled.rego

```diff
@@ -0,0 +1,52 @@
+# METADATA
+# title: "Verify S3 bucket server-side-encryption is enabled"
+# description: "This policy audits S3 buckets for Server-Side Encryption (SSE) to enhance security by encrypting data at rest, ensuring protection for stored files and directories transparently to users."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-s3-bucket.html
+# custom:
+#   id: CB_CFT_010
+#   severity: LOW
+package lib.cloudformation.CB_CFT_010
+import future.keywords.in
+
+resource := "AWS::S3::Bucket"
+
+is_valid {
+    some resources in input.Resources
+    resources.Type == resource
+}
+
+pass[resources] {
```

Comment:
update logic for pass
pass[encryption] {
    is_valid
    some resources in input.Resources
    some encryption in resources.Properties.BucketEncryption.ServerSideEncryptionConfiguration
    encryption.ServerSideEncryptionByDefault.SSEAlgorithm = "AES256"
}